tests/metrics/test_current_metrics.py

```diff
@@ -116,8 +116,8 @@ def test_current_metrics(
     num_actual_calls = 0
 
     # Need to filter out non-metrics-related calls
-    for call in mock_logger_destination.log_data.mock_calls:
-        data = call[1][2]
+    for call in mock_logger_destination.log_metrics.mock_calls:
+        data = call[1][0]
```

Comment:
can you put a comment here? what exactly is the indexing doing and why did it change? (Mostly so test is more readable)

Reply:
fixed here: 
21f75e4 to be more readable